integration/commands_administration_compat_test.go

```diff
@@ -57,6 +58,13 @@ func TestListCollectionsCompat(t *testing.T) {
 	err = compat.All(ctx, &compatRes)
 	require.NoError(t, err)
 
+	compatNames := make([]string, len(compatRes))
+	for i, doc := range compatRes {
+		compatNames[i] = doc.Map()["name"].(string)
+	}
+
+	require.True(t, slices.IsSorted(compatNames), "compat collections are not sorted")
```

Comment:
We should not check that MongoDB collections are sorted because they are not:

> The `listCollections` command returns a document that contains an **unsorted** list of all collections and views in the database

From https://www.mongodb.com/docs/manual/reference/command/listCollections/